front/src/components/UI/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import { PackageData } from '../../App/Help/Help';
+import EditTabsButton from './EditTabsButton/EditTabsButton';
+import { AddStyled, ContentStyled, SingleTab, TabsStyled, TitleStyled } from "./Tabs.styled";
+import './Tabs.styled.ts';
+
+const Tabs = (props: { packageData: PackageData[], addElementHandler: Function }) => {
+
+  const addContentHandler = (elements: PackageData[]): PropTypes.ReactNodeArray => {
+    const elementHTMLarray = [];
+
+
+    for (let i = 0; i < elements.length; i++) {
+      let {id, name, content} = elements[i];
+      
+      //single tab Component
+      elementHTMLarray.push(
+          <SingleTab>
+            <TitleStyled>{name}</TitleStyled>
+            <ContentStyled> 
+              {content}
+            </ContentStyled>
+            <Link to={{pathname: '/edit', state: {element : props.packageData[i]}}}>
```

Comment:
nie wpisujemy na sztywno takich danych jak pathanme. Już masz to w 2 miejscach i w obu trzeba zmienić jeśli wprowadzimy zmianę w nazwie. Zamiast tego w App.tsx jest enum w którym można to zadeklarować daną ścieżkę i dzięki temu można jej użyć wiele razy, a edycja w przyszłości nie spowoduje rozjazdów.